src/validators/dict.rs

```diff
@@ -87,6 +87,10 @@ impl Validator for DictValidator {
     fn get_name(&self) -> &str {
         &self.name
     }
+
+    fn supports_partial(&self) -> bool {
+        true
+    }
```

Comment:
What about children / nested validators, e.g. unions etc, do they need to have this implemented? `with-default`  / `nullable`?

If not, should this be `const SUPPORTS_PARTIAL: bool`?

Reply:
I don't think children are a consideration here.

I tried `const SUPPORTS_PARTIAL: bool`, but it breaks `enum_dispatch`